### ISSUE TITLE
pc (pkg-config) file missing space around ">=" sign

### ISSUE THREAD

**Issue by andreadelprete** [stack-of-tasks/pinocchio#940]
Installing the latest version of pinocchio (devel) from source I got an apparently faulty pc file, which contained the following line:
```
Requires: eigen3>=3.0.5, urdfdom>=0.2.0
```
The lack of a "space" before and after ">=" gave problem to pkg-config on my machine (Ubuntu 16.04). For instance:
```
$ pkg-config --print-requires pinocchio
Package eigen3>=3.0.5 was not found in the pkg-config search path.
Perhaps you should add the directory containing `eigen3>=3.0.5.pc'
to the PKG_CONFIG_PATH environment variable
Package 'eigen3>=3.0.5', required by 'pinocchio', not found
```
Adding the missing spaces solved the problem.

**Comment by nim65s** [comment]
Hi @andreadelprete,

This has already been reported by @wxmerkt  here: https://github.com/stack-of-tasks/pinocchio/pull/933#issuecomment-551333575 ; and @sbarthelemy  is on it.

**Comment by nim65s** [comment]
You can just temporarily and locally revert 2adf778 to fix this, if necessary

**Comment by andreadelprete** [comment]
ok thanks!

**Comment by jcarpent** [comment]
@andreadelprete https://github.com/jrl-umi3218/jrl-cmakemodules/pull/278 should fix the current issue.

**Comment by jcarpent** [comment]
@sbarthelemy I have proposed a fix because it is affecting current users. Sorry for the bypass.

**Comment by jcarpent** [comment]
Solved by #941.
